gapic-generator/lib/gapic/presenters/gem_presenter.rb

```diff
@@ -134,11 +134,11 @@ def api_id
       end
 
       def free_tier?
-        gem_config(:free_tier) ? true : false
+        gem_config(:free_tier) == "true"
       end
 
       def yard_strict?
-        gem_config(:yard_strict) ? true : false
```

Comment:
I'd add some comments or a helper function clarifying the intent here -- it seems that the intent is for `:free_tier` to default to false, and for `:yard_strict` to default to true, but is it really?

Reply:
👍